### PR TITLE
Accessibility support

### PR DESCRIPTION
**Pull request by WilcoFiers**
Updated accessibility support

### CODE REVIEW

File: act-rules-format.bs [hunk 1]
```diff
@@ -195,6 +165,28 @@ If either one of these passed, the rule has passed. Only if both fail, does that
...
+In accessibility testing, there are often multiple ways to solve the same problem. For instance, in HTML tables, header cells can be indicated through the `scope` attribute, by using the `headers` attributes, or by using ARIA labels. All of these separate techniques could be described in one big rule. But this creates a large, and often difficult to maintain rule. To ensure rules are kept small and atomic, they SHOULD be put into a rule-group instead.
+
+To meet the accessibility requirement, only one rule in a rule-group has to pass. This way, for our table example, one could write three separate rules, one for scope, one for headers+id and one ARIA labels). If a table meets any of these rules, it automatically passes the group, and the failed results of the other rules can be ignored.
+
+Elements can pass a rules group if at least one rule within that group selected that element, and returned a passed result. Likewise, an element will fail the rule group, if any rule in the group selected it, but none of them passed it. In the example of HTML Tables, passing the table scope rule, and railing the aria label and headers+id rule, would mean the element passed this group of rules.
```
Comment (kasperisager) [inline act-rules-format.bs]: "railing" -> "failing"

Comment (kasperisager) [inline act-rules-format.bs]: I'd probably also remove the before "would mean".

File: act-rules-format.bs [hunk 2]
```diff
@@ -195,6 +165,28 @@ If either one of these passed, the rule has passed. Only if both fail, does that
...
+
+Note that rules in a group MAY have different selectors. Because of this, not every element selected within the group, is tested by every rule. Rules MAY also be disabled during a test run due to accessibility support concerns. See [Accessibility support](#acc-support) for details.
```
Comment (kasperisager) [inline act-rules-format.bs]: No comma before "is tested".

File: act-rules-format.bs [hunk 3]
```diff
@@ -195,6 +165,28 @@ If either one of these passed, the rule has passed. Only if both fail, does that
...
+
+
+Accessibility Support {#acc-support}
+===========================
+
+ACT Rules are designed to test good and bad uses of a web technology. Not every part of a web technology is implemented in all assistive technologies a website may need to support. This is known as [Accessibility Supported in WCAG 2.0](https://www.w3.org/TR/WCAG20/#accessibility-supporteddef). Because of this, some rules should not be used in a particular test scenario. For instance, rules related to ARIA, should not be used on a site that is supposed to work for assistive technologies that have no ARIA support.
```
Comment (kasperisager) [inline act-rules-format.bs]: No comma before "should not".

File: act-rules-format.bs [hunk 4]
```diff
@@ -195,6 +165,28 @@ If either one of these passed, the rule has passed. Only if both fail, does that
...
+
+Even within a rules group, certain rules may not be used. This leaves one fewer solution for passing that particular rule group. Because of this, ACT Rules has to be atomic, and so it MUST NOT rely on other rules to be part of the same test scenario.
```
Comment (kasperisager) [inline act-rules-format.bs]: "ACT Rules _have_ to be"

File: act-rules-format.bs [hunk 5]
```diff
@@ -195,6 +165,28 @@ If either one of these passed, the rule has passed. Only if both fail, does that
 </div>
 
 
+Rule Grouping {#grouping}
+===================
+
+In accessibility testing, there are often multiple ways to solve the same problem. For instance, in HTML tables, header cells can be indicated through the `scope` attribute, by using the `headers` attributes, or by using ARIA labels. All of these separate techniques could be described in one big rule. But this creates a large, and often difficult to maintain rule. To ensure rules are kept small and atomic, they SHOULD be put into a rule-group instead.
+
+To meet the accessibility requirement, only one rule in a rule-group has to pass. This way, for our table example, one could write three separate rules, one for scope, one for headers+id and one ARIA labels). If a table meets any of these rules, it automatically passes the group, and the failed results of the other rules can be ignored.
```
Comment (annethyme) [inline act-rules-format.bs]: ... or in this case only 2 rules have to pass: 
"2.4.5 - Multiple Ways": "Using two or more of the following techniques:"

Comment (moekraft) [inline act-rules-format.bs]: Need to be consistent with the use of "rule-group." Later we do not hyphenate it. I don't think it need hyphenation.

Comment (moekraft) [inline act-rules-format.bs]: Second to last sentence in first paragraph, "But this creates a <moe>complex rule that is difficult to maintain</moe>"

**Comment by WilcoFiers** [comment]
Updated according to the provided feedback.

**Pull request merged**